lib/cadence/testing/cadence_override.rb

```diff
@@ -101,8 +101,6 @@ def start_locally(workflow, *input, **args)
           execution, workflow_id, run_id, workflow.disabled_releases, metadata
         )
 
-        Cadence::ThreadLocalContext.set(context)
```

Comment:
@anthonydmitriyev are you okay with me keeping this change? Since the context gets set in execute_in_context call

Reply:
Yes, absolutely that should work!